src/Kiota.Builder/Writers/Python/CodeClassDeclarationWriter.cs

```diff
@@ -25,23 +25,18 @@ public override void WriteCodeElement(ClassDeclaration codeElement, LanguageWrit
                 _codeUsingWriter.WriteDeferredImport(parentClass, codeElement.Inherits.Name, writer);
             foreach (var implement in codeElement.Implements)
                 _codeUsingWriter.WriteDeferredImport(parentClass, implement.Name, writer);
-
+            if (parentClass.IsOfKind(CodeClassKind.Model) | parentClass.Parent is CodeClass)
```

Comment:
Any reason to use the single pipe? This will force the evaluation of both conditions when if you only need one to be true..
```suggestion
            if (parentClass.IsOfKind(CodeClassKind.Model) || parentClass.Parent is CodeClass)
```